src/main/webapp/college-info.html

```diff
@@ -9,13 +9,17 @@
   </head>
   <body>
     <header>
-      <h1>Black Student Insight</h1>
-      <input type="text" id="school-search">
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution">
       <search-button onclick="loadSchoolInfo()">Search</search-button>
     </header>
     <main>
-      <h1 id="school-name"></h1>
+      <div class="topnav">
+        <a class="active" href="College Info.html">School Info</a>  
+        <a href="comments.html">Student Reviews</a>
```

Comment:
Is this replacing the reviews button that was originally at the bottom of the page? The way it will work to load the correct comments is delete the href attribute and add id="reviews-button". The href gets added later in javascript

Reply:
Yes, it's supposed to replace the reviews button and I'll make those changes